charts/hyperflow-observability/templates/metric-rules.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: hyperflow-rules
+  labels:
+    app: kube-prometheus-stack
+spec:
+  groups:
+    - name: hyperflow-deployment-metrics
+      interval: 1s
+      rules:
+        - record: hyperflow_deployment_status_replicas_available
+          expr: |
+            kube_deployment_status_replicas_available
+            * on(namespace, deployment) group_left(label_origin)
+            kube_deployment_labels{label_origin="hyperflow"}
+    - name: node_cpu_usage
+      interval: 5s
+      rules:
+        - record: node_cpu_usage_percent
```

Comment:
@balis I would recommend a 1-2 sentence per metric explaining what it does, so next people can have a less steep learning curve